consoles/localXvnc.pm

```diff
@@ -27,6 +27,18 @@ require IPC::System::Simple;
 use Socket;
 use File::Which;
 
+sub sshCommand {
+    my ($self, $username, $host, $gui) = @_;
+
+    my $sshopts = "-o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -o PubkeyAuthentication=no $username\@$host";
+
+    if ($gui) {
+        $sshopts = "-X $sshopts";
+    }
+
+    return "ssh $sshopts; read";
```

Comment:
Moving the function makes sense. When moving it anyways you can also get rid of the blank lines.

Reply:
done. To go even further see #1531